Client/game_sa/CHandlingManagerSA.cpp

```diff
@@ -120,10 +120,17 @@ static __declspec(naked) void Hook_Calculate()
     }
 }
 
-static bool IsVehicleModel(eVehicleTypes eModel)
+static bool IsVehicleModel(std::uint32_t uiModel)
 {
-    const auto pModelInfo = pGame->GetModelInfo(eModel);
-    return pModelInfo && pModelInfo->IsVehicle();
+    try
+    {
+        const CModelInfo* pModelInfo = pGame->GetModelInfo(uiModel);
+        return pModelInfo && pModelInfo->IsVehicle();
+    }
+    catch (...)
+    {
+        return false;
+    }
```

Comment:
can be noexcept if try-catch

Reply:
?

```
static bool IsVehicleModel(std::uint32_t model) noexcept
{
    const CModelInfo* pModelInfo = pGame->GetModelInfo(model);
    return pModelInfo && pModelInfo->IsVehicle();
}
```

Reply:
I didn't understand you much earlier, okay